src/map/clif.c

```diff
@@ -1417,10 +1437,20 @@ static void clif_set_unit_walking(struct block_list *bl, struct map_session_data
 	p.font = (sd) ? sd->status.font : 0;
 #endif
 #if PACKETVER >= 20120221
-	if (battle_config.show_monster_hp_bar && bl->type == BL_MOB && status_get_hp(bl) < status_get_max_hp(bl)) {
+	const struct mob_data *md = BL_UCCAST(BL_MOB, bl);
```

Comment:
md already exists below in clif.c:1458. you cant define same variable name more than once in one code block

Reply:
@4144 Pushed commit for your requested changes, please check. Thank you! :)